components/lib/calendar/Calendar.js

```diff
@@ -480,6 +480,7 @@ export const Calendar = React.memo(
             const currentViewDate = getViewDate();
             let newViewDate = cloneDate(currentViewDate);
```

Comment:
Can i simply use new Date() ?

Reply:
No this looks good if you look down in the calendar you will see we do this...

```js
currentDate.setDate(1); // #2948 always set to 1st of month
```

So clearly this spot was missed.